# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.h
# Commit: 96bd98319c62
# Full Hash: 96bd98319c6242a9b05dc4723b928e88537424e1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-26 03:24:37
# Regressor Bug: 1652667
# File Overlap Count: 1
# Description:
#   Bug 1652667 - P1: Add telemetry for when an HTTPSSVC record is received r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87844
# ==============================================================================

diff -r b61a789151df -r 96bd98319c62 netwerk/protocol/http/nsHttpChannel.h
--- a/netwerk/protocol/http/nsHttpChannel.h	Tue Aug 25 17:29:21 2020 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.h	Tue Aug 25 17:29:32 2020 +0000
@@ -739,6 +739,9 @@
 
   uint32_t mUseHTTPSSVC : 1;
   uint32_t mWaitHTTPSSVCRecord : 1;
+  // Only set to true when we receive an HTTPSSVC record before the transaction
+  // is created.
+  uint32_t mHTTPSSVCTelemetryReported : 1;
 
   // The origin of the top window, only valid when mTopWindowOriginComputed is
   // true.